cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/cmd/InitCommand.java

```diff
@@ -144,12 +140,13 @@ public void execute() {
                     } else {
                         out.println("Invalid input");
                     }
-
-                    first = false;
                 } while (!validInput);
 
                 out.print("\n");
             } else {
+                manifest = new Manifest();
+                manifest.setName(guessOrgName());
+                manifest.setVersion(DEFAULT_VERSION);
```

Comment:
What would happen if theres already a Ballerina.toml file ?

Reply:
If there's already a Ballerina.toml inside the project, it is not created again